driver/controller.go

```diff
@@ -107,10 +105,10 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		if v.Name == req.Name {
 			return &csi.CreateVolumeResponse{
 				Volume: &csi.Volume{
-					VolumeId: exoscaleID(d.zone, v.ID),
+					VolumeId: v.ID.String(),
```

Comment:
Will this cause issues for customer that already use the csi ? 

Their volumeID will be in the form `shortZone/uuid` while new ones will be just `uuid`

What about keeping the `shortZone/uuid` but discarding the zone part for the moment ? That way adding back multizone support doesn't require a migration either